ZelBack/src/services/appsService.js

```diff
@@ -8347,13 +8401,25 @@ async function checkAndNotifyPeersOfRunningApps() {
         installedAndRunning.push(app);
       }
     });
-    // eslint-disable-next-line no-restricted-syntax
-    for (const application of installedAndRunning) {
-      log.info(`${application.name} is running properly. Broadcasting status.`);
-      try {
+
+    const syncStatus = daemonServiceMiscRpcs.isDaemonSynced();
+    const daemonHeight = syncStatus.data.height || 0;
+
+    if (daemonHeight >= config.fluxapps.apprunningv2 && installedAndRunning.length > 1) {
```

Comment:
to be simplified to not have repetitive code

Reply:
ohh, I have missed this one... let me see what can I do